alternatives/drivers/BK1_tinytc.cc

```diff
@@ -94,88 +116,140 @@ void run_test(
     const size_t size_basis1 = basis1.size();
     const size_t size_basis2 = basis2.size();
 
-    T *d_JxW    = sycl::malloc_device<T>(size_JxW, queue);
-    T *d_in     = sycl::malloc_device<T>(size_in, queue);
-    T *d_out    = sycl::malloc_device<T>(size_out, queue);
+    T *d_JxW = sycl::malloc_device<T>(size_JxW, queue);
+    T *d_in = sycl::malloc_device<T>(size_in, queue);
+    T *d_out = sycl::malloc_device<T>(size_out, queue);
     T *d_basis0 = sycl::malloc_device<T>(size_basis0, queue);
     T *d_basis1 = sycl::malloc_device<T>(size_basis1, queue);
     T *d_basis2 = sycl::malloc_device<T>(size_basis2, queue);
 
-    queue.copy(in.data(),d_in,size_in);
-    queue.copy(JxW.data(),d_JxW,size_JxW);
-    queue.copy(basis0.data(),d_basis0,size_basis0);
-    queue.copy(basis1.data(),d_basis1,size_basis1);
-    queue.copy(basis2.data(),d_basis2,size_basis2);
+    queue.copy(in.data(), d_in, size_in);
+    queue.copy(JxW.data(), d_JxW, size_JxW);
+    queue.copy(basis0.data(), d_basis0, size_basis0);
+    queue.copy(basis1.data(), d_basis1, size_basis1);
+    queue.copy(basis2.data(), d_basis2, size_basis2);
     queue.wait_and_throw();
 
-//    const size_t localSize = std::min(nq0 * nq1 * nq2, (int) threadsPerBlock);
-//    const size_t globalSize = numBlocks * localSize;
-
-//    const sycl::nd_range<1> kernelRange{{globalSize},{localSize}};
-
-//    const std::array<unsigned int,3> nms{nm0,nm1,nm2};
-//    const std::array<unsigned int,3> nqs{nq0,nq1,nq2};
-
-//    const std::array<int,3> nq_sf{nq0,nq1,nq2};
-
     // Performance in GDoF/s
     auto dof_rate = [=](double elapsed) {
         return 1.0e-9 * nelmt * nm0 * nm1 * nm2 / elapsed;
     };
+    auto byte_rate = [=](double elapsed) {
+        return 1.0e-9 * sizeof(float) * nelmt *
+               (2 * nm0 * nm1 * nm2 + nq0 * nq1 * nq2) / elapsed;
+    };
```

Comment:
This now includes the data for `in`, `out`, and `JxW` right? 

Since `nq0*nq1*nq2 = (nm0+1)*(nm1+1)*(nm2+1)`, for `p = 8, 8^3 = 512, 8^2 = 64`, the quadratic terms aren't negligible. 

cc @sydn00, @kronbichler  I think we should fix this in the remaining drivers too. It will make a fairer comparison.